python/segyio/trace.py

```diff
@@ -84,6 +84,10 @@ class Trace(Sequence):
     >>> tr = tr - 100
     >>> avg = numpy.average(tr)
 
+    Perform some seismic processing on a trace. E.g resample from 2ms spacing to 4ms spacing (note there is no anti-alias filtering in this example):
```

Comment:
This line is a bit long - please reflow it to be 79 characters inclusive.

Reply:
this has been done with the next commit

Reply:
Thank you, I appreciate it.